lib/file/widgets/upload_fab.dart

```diff
@@ -14,7 +14,7 @@ class UploadFab extends StatelessWidget {
   Widget build(BuildContext context) {
     return EntityBuilder<User>(
       id: services.storage.userId,
-      builder: handleEdgeCases((context, user, _) {
+      builder: handleLoadingError((context, user, _) {
         if (user == null || !user.hasPermission(Permission.fileStorageCreate)) {
```

Comment:
Loading is already handeled.
```suggestion
        if (!user.hasPermission(Permission.fileStorageCreate)) {
```